otel-infrastructure-collector/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: opentelemetry-collector
+  repository: https://open-telemetry.github.io/opentelemetry-helm-charts
+  version: 0.40.2
+digest: sha256:873e3ff92e016940de493bea71787dc8792e37ee3e4bd59ec2712d979bef0e96
+generated: "2022-11-28T10:06:47.507682498+02:00"
```

Comment:
Lets delete this file like we did for the daemonset chart, or add 'helm dependency update' to the workflow, so we wont have to change this manually